.circleci/config.yml

```diff
@@ -179,103 +201,123 @@ jobs:
       - restore_cache: *restore_cache
       - run: *check_cache
       - run: *install_cached_packages
-      - run:
-          command: |
-            export openqa_unstable_tests="$(cat .circleci/unstable_tests.txt)"
-            if [ ! -z "$openqa_unstable_tests" ]; then
-              eval "$(t/test_postgresql | grep TEST_PG=)" && PERL5LIB=lib prove -v $openqa_unstable_tests || echo FAIL
-            fi
+      - run: make checkstyle
 
-  testapi:
-    docker:
-      - <<: *base
-    steps:
-      - checkout
-      - run: *chown_hack_for_cache
-      - restore_cache: *restore_cache
-      - run: *check_cache
-      - run: *install_cached_packages
-      - run:
-          command: |
-            eval "$(t/test_postgresql | grep TEST_PG=)" && PERL5LIB=lib prove -v t/api/*.t
+  t.ui:
+    <<: *test-template
+    environment:
+      TEST_PART=ui
+
+  t.unstable:
+    <<: *test-template
+    environment:
+      TEST_PART=unstable
+
+  t.api:
+    <<: *test-template
+    environment:
+      TEST_PART=api
+
+  t.developer:
+    <<: *test-fullstack-template
+    environment:
+      TEST_PART=developer-fullstack
+
+  t.scheduler:
+    <<: *test-fullstack-template
+    environment:
+      TEST_PART=scheduler-fullstack
 
-  testfullstack:
+  report.codecov:
     docker:
       - <<: *base
     steps:
       - checkout
-      - run: echo CIRCLE_WORKFLOW_ID=$CIRCLE_WORKFLOW_ID
-      - run: *chown_hack_for_cache_integration
-      - restore_cache: *restore_cache
-      - restore_cache: *restore_integration_cache
-      - run: *check_cache
-      - run: *install_cached_packages
-      - run: *build_autoinst
-      - run:
-          command: |
-            eval "$(t/test_postgresql | grep TEST_PG=)" && PERL5LIB=lib FULLSTACK=1 prove -v t/full-stack.t
-
-  testdeveloperfullstack:
+      - attach_workspace:
+          at: .
+      - run: sudo zypper ar -f https://download.opensuse.org/repositories/devel:/languages:/perl/openSUSE_Leap_15.1/ devel-perl
```

Comment:
All requirements we potentially have should be covered in devel:openQA and it's referred projects. What package is missing?

Reply:
Devel-Cover-1.29 cannot merge DBs properly, I've submitted https://build.opensuse.org/request/show/735614